deploy/lib/parca-agent/parca-agent.libsonnet

```diff
@@ -303,12 +311,28 @@ function(params) {
                 name: 'tmp',
                 emptyDir: {},
               },
+              // Needed for reading the container runtime metadata.
               {
                 name: 'run',
                 hostPath: {
                   path: '/run',
                 },
               },
+              // Needed for reading kernel configuration.
+              {
+                name: 'boot',
+                hostPath: {
+                  path: '/boot',
```

Comment:
It slightly bothers me that Kubernetes does not have way to have optional "if exists" host volumes. Here, if `/boot` does not exist on the node, it will be created which can be considered as left-over garbage if the agent is uninstalled from the cluster, so it may not be a questionable default

Reply:
No, you're right, not a big issue